Integration test failure

After merging the basepom change, I get repeatably the following exception on verification of the integration test:

```
[INFO] --- maven-failsafe-plugin:2.20:verify (default) @ runtime ---
[INFO] Failsafe report directory: /Users/roland/Development/syndesis/syndesis-rest/runtime/target/failsafe-reports
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Syndesis REST :: Parent ............................ SUCCESS [  5.836 s]
[INFO] Syndesis REST :: Connector Catalog ................. SUCCESS [  9.660 s]
[INFO] Syndesis REST :: Core .............................. SUCCESS [  9.099 s]
[INFO] Syndesis REST :: Model ............................. SUCCESS [ 20.255 s]
[INFO] Syndesis REST :: DAO ............................... SUCCESS [ 20.372 s]
[INFO] Syndesis REST :: GitHub Access ..................... SUCCESS [ 11.987 s]
[INFO] Syndesis REST :: OpenShift Services ................ SUCCESS [ 15.896 s]
[INFO] Syndesis REST :: Project Generator ................. SUCCESS [ 20.251 s]
[INFO] Syndesis REST :: Controllers ....................... SUCCESS [ 16.948 s]
[INFO] Syndesis REST :: JsonDB ............................ SUCCESS [ 27.043 s]
[INFO] Syndesis REST :: Model2 ............................ SUCCESS [ 25.239 s]
[INFO] Syndesis REST :: Verifier .......................... SUCCESS [ 16.220 s]
[INFO] Syndesis REST :: REST .............................. SUCCESS [ 39.890 s]
[INFO] Syndesis REST :: Runtime ........................... FAILURE [04:26 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 08:25 min
[INFO] Finished at: 2017-06-06T15:31:53+02:00
[INFO] Final Memory: 246M/1725M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:2.20:verify (default) on project runtime: There was a timeout or other error in the fork -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:2.20:verify (default) on project runtime: There was a timeout or other error in the fork
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:213)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:309)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:194)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:107)
```

--- a/runtime/pom.xml
+++ b/runtime/pom.xml
@@ -233,6 +233,7 @@
                 <!-- workaround suggested in https://github.com/spring-projects/spring-boot/issues/6254 -->
                 <additionalClasspathElement>${project.build.outputDirectory}</additionalClasspathElement>
               </additionalClasspathElements>
+              <forkedProcessTimeoutInSeconds>150</forkedProcessTimeoutInSeconds>
               <systemProperties>
                 <keycloak.http.port>${keycloak.http.port}</keycloak.http.port>
               </systemProperties>

